app/controllers/password_controller.rb

```diff
@@ -41,6 +41,12 @@ def forgot_form
     @form = ForgottenPasswordForm.new
   end
 
+  def reset_user_password_send_code
```

Comment:
a tiny thing, the name is a bit vague/confusing... would something like `force_user_reset_password` be more descriptive?